products/gitlab.md

```diff
@@ -19,11 +19,17 @@ auto:
     regex: '^v?(?<major>[1-9]\d*)\.(?<minor>\d+)\.(?<patch>\d+)-ee?$'
 
 
-# EOL of R = releaseDate(R+3)
-# Support of R = releaseDate(R+1)
-# Releases are made on third Thursday of every month (https://about.gitlab.com/blog/2023/09/18/gitlab-release-date-change/).
+# support(x) = releaseDate(x+1)
+# eol(x) = releaseDate(x+3)
+# Upcoming release dates are available on https://about.gitlab.com/releases/.
```

Comment:
We should put that link to the description at the end. WDYT?

Reply:
Done.